### PR TITLE
Feature/implement notifications

### PR DESCRIPTION
**Pull request by AndreiMiron-hub**

### CODE REVIEW

File: source/Assist.Jully.2022.BE2/Assist.July._2022.BE2.Application/Dtos/NotificationSettingsDtos/AddNotificationDto.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿namespace Assist.July._2022.BE2.Application.Dtos.NotificationSettingsDtos
+{
+    public class AddNotificationDto
+    {
+        public Guid UserId { get; set; }
+        public DiscountNotificationDto discountNotificationDto { get; set; }
+        public ListingsNotificationDto listingsNotificationDto { get; set; }
```
Comment (SebastianFulgaAssist) [inline source/Assist.Jully.2022.BE2/Assist.July._2022.BE2.Application/Dtos/NotificationSettingsDtos/AddNotificationDto.cs:7]: PascalCase please

File: source/Assist.Jully.2022.BE2/Assist.July._2022.BE2.Application/Dtos/NotificationSettingsDtos/DiscountNotificationDto.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Assist.July._2022.BE2.Application.Dtos.NotificationSettingsDtos
+{
+    public class DiscountNotificationDto
+    {
+        public bool email { get; set; } = false;
```
Comment (SebastianFulgaAssist) [inline source/Assist.Jully.2022.BE2/Assist.July._2022.BE2.Application/Dtos/NotificationSettingsDtos/DiscountNotificationDto.cs:5]: PascalCase

File: source/Assist.Jully.2022.BE2/Assist.July._2022.BE2.Application/Dtos/NotificationSettingsDtos/NewsNotificationDto.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Assist.July._2022.BE2.Application.Dtos.NotificationSettingsDtos
+{
+    public class NewsNotificationDto
+    {
+        public bool email { get; set; } = false;
```
Comment (SebastianFulgaAssist) [inline source/Assist.Jully.2022.BE2/Assist.July._2022.BE2.Application/Dtos/NotificationSettingsDtos/NewsNotificationDto.cs:5]: You can also do the initializations into a Default Constructor

File: source/Assist.Jully.2022.BE2/Assist.July._2022.BE2.Application/Dtos/NotificationSettingsDtos/NotificationResponseDto.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿namespace Assist.July._2022.BE2.Application.Dtos.NotificationSettingsDtos
+{
+    public class NotificationResponseDto
+    {
+        public DiscountNotificationDto? discountNotificationDto { get; set; }
+        public ListingsNotificationDto? listingsNotificationDto { get; set; }
```
Comment (SebastianFulgaAssist) [inline source/Assist.Jully.2022.BE2/Assist.July._2022.BE2.Application/Dtos/NotificationSettingsDtos/NotificationResponseDto.cs:6]: PascalCase

**Review by HincuAndreiAssist — APPROVED** [review]

**Review by SebastianFulgaAssist — APPROVED** [review]

**Pull request merged**